src/components/Confirm/style.css

```diff
@@ -10,8 +10,9 @@
 }
 
 .livenessVideo {
-  width: 100%   !important;
-  height: auto  !important;
+  width: 100%;
+  height: 100%;
```

Comment:
If we use `object-fit: contain` I can see that the video controls do not resize to sit at the bottom of the video stream. I think we should use `height: auto; object-fit: cover;`

Reply:
Here is a screenshot of how it looks at the moment on Samsung Galaxy S8. See the gradient at the bottom
![screenshot_20180821-153949_chrome](https://user-images.githubusercontent.com/7127427/44408842-19e65d00-a559-11e8-8ac5-73e80c80783e.jpg)

Reply:
Thanks @stefaniacardenas .

I see it as 3 different options:

1- Video container height is 300px container, video takes 100% of that and displays as `object-fit: contain` (the whole video is displayed inside those 300px, unless proportions match exactly those of the video there will be some blank space, displayed as a gradient on Chrome)
2- Video container height is 300px container, video takes 100% of that and displays as `object-fit: cover` (no white space, but some of the video is not visible)
3- Height set to `auto`: no empty space, but video might push buttons down and make the content overflow

Reply:
@stefaniacardenas, as I understand #3? If so, then there is also another fix to be made, since the preview is contained in an absolutely positioned element, and that is what seems to be the source of this bug.

Reply:
What about adding `max-height: 100%`?

Reply:
1)
![image](https://user-images.githubusercontent.com/1504258/44414271-a8fd7000-a56d-11e8-9f18-0083f0440b17.png)

2)
![image](https://user-images.githubusercontent.com/1504258/44414278-ad298d80-a56d-11e8-9dc0-34a990294fc8.png)

3)
![image](https://user-images.githubusercontent.com/1504258/44414286-b31f6e80-a56d-11e8-9c5f-e9bc2fdffd8b.png)

3-b)
![image](https://user-images.githubusercontent.com/1504258/44414344-cfbba680-a56d-11e8-874d-950821d6e1d7.png)

Reply:
@stefaniacardenas: `max-height: 100%` doesn't really affect how it's displayed 😔

Reply:
@seewah @stefaniacardenas , as I understand we want to use the same approach used for the selfie screen, and that is contained within a limited height:

![image](https://user-images.githubusercontent.com/1504258/44414706-c121bf00-a56e-11e8-8985-234354662c1c.png)

In my opinion that is closer to `object-fit: contain; height: 100%`

Reply:
Yes, maybe that's the way to go. And if the video controls look weird we could remove them and trigger autoplay. Any thoughts  @rfreitas  @seewah?